leku/src/main/java/com/schibstedspain/leku/geocoder/GeocoderPresenter.java

```diff
@@ -56,6 +59,7 @@ public void getFromLocationName(String query) {
         .subscribeOn(Schedulers.newThread())
         .observeOn(scheduler)
         .retry(RETRY_COUNT)
+        .onErrorResumeNext(apiInteractor.getFromLocationName(query))
```

Comment:
i don't. think this is the correct place to do this logic.

presenter should be agnostic on where to get address.

presenter receives a UseCase, and it has two (or N) apis (repository?).